nilearn/decoding/decoder.py

```diff
@@ -1671,21 +1626,7 @@ def __sklearn_tags__(self):
         See the sklearn documentation for more details on tags
         https://scikit-learn.org/1.6/developers/develop.html#estimator-tags
         """
-        # TODO
-        # get rid of if block
-        # bumping sklearn_version > 1.5
-        # see https://github.com/scikit-learn/scikit-learn/pull/29677
-        tags = super().__sklearn_tags__()
-        if SKLEARN_LT_1_6:
-            tags["multioutput"] = True
-            return tags
-
-        from sklearn.utils import RegressorTags
-
-        tags.estimator_type = "regressor"
-        tags.regressor_tags = RegressorTags()
-
-        return tags
+        return super().__sklearn_tags__()
```

Comment:
The same here.

Reply:
ok turns out that it is needed otherwise the DecoderRegressor will only get the tags from _RegressorMixin